qep-321-contributor-recognition.md

```diff
@@ -0,0 +1,80 @@
+# QGIS Enhancement 321: Contributor recognition and commit rights policies
+
+The QGIS project splits the concept of "notable developer recognition" from those with commit rights to the code
+repository. This is done in order to maintain good security practice for the code repository, with only those who
+are currently actively contributing to QGIS code having push/merge rights to the repository.
+
+Accordingly, there are two different types of QGIS developer status:
+
+## 1. QGIS Core Contributors
+
+"Core contributor" status is a lifetime status granted to developers who have demonstrated ongoing commitment and
+excellence in QGIS development.
+
+In order to be eligible, a developer must have:
+
+- 1.0.1. A history of high-quality code contributions to QGIS for a period of at least 12 months
+- 1.0.2. Demonstrated a high level of knowledge of c++, Python, the QGIS API, and QGIS development processes and practices
+- 1.0.3. Demonstrated a willingness to work in a friendly, collaborative manner and respond positively to change requests
+  and suggestions from other core contributors
+
+(While the project recognises the **immense** value in non-code contributions to QGIS, this status is designed exclusively for
+contributors of code.)
+
+## 1.1. Process for Nomination
+
+TODO
+
+# 2. Commit Rights
+
+This group of developers are those with write access to the main QGIS code repository. Unlike the "core contributor"
+status, this is a transient right and reflects only a developer's **current** activity within the QGIS project.
+**It is not a reflection of a developer's abilities, trustworthiness or skill, and is considered purely an
```

Comment:
This is not really true : to be a committer, you need to be trusted, and you also need to have the ability to understand contribution's processes.